### PR TITLE
Homework20

### PR DESCRIPTION
**Pull request by DanilKrivulin**

### CODE REVIEW

File: src/test/java/HW20.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import com.fasterxml.jackson.databind.ser.Serializers;
+import org.testng.annotations.Test;
+
+public class HW20 extends BaseTest {
```
Comment (Berichicko) [inline src/test/java/HW20.java:4]: **According to the Java style guide:** 

1. Class names should be nouns, in mixed cases with the first letter of each internal word capitalized. Interfaces names should also be capitalized just like class names.
2. Use whole words and must avoid acronyms and abbreviations.

For example `public class HomeWork20`

File: src/test/java/HW20.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    @Test
+    public void createPlaylist2() {
```
Comment (Berichicko) [inline src/test/java/HW20.java:6]: Adhere to the following guidelines to write expressive, easily readable test names:

- `createPlaylistTest()` is the name of the method you are testing.
- avoid "magic" number ( createPlaylistTest1, reatePlaylistTest2...)

**Comment by Berichicko** [comment]
Hello 
Homework 20 is cheeked!
Fantastic job! Magnificent! 
💖Best wishes, the team HW (Alex,  aliaksandr@testpro.io)!💖


image

File: src/test/java/HW20.java (hunk 2 above)
Comment (DanilKrivulin) [reply src/test/java/HW20.java:6]: Ok, thank you

**Pull request closed without merging**